templates/csharp/sso-tab/teamsapp.yml.tpl

```diff
@@ -82,18 +82,21 @@ provision:
     with:
       # Path to manifest template
       manifestPath: ./appPackage/manifest.json
+
```

Comment:
Why do we add so many blank lines? Is there common rule?

Reply:
This is because I saw that blank lines in constraints were not regular, so I standardized them.

Reply:
You can refer to node.js templates that we do not add blank line between teams App related actions.